javascript/javascript2/week3/fastestPresser.js

```diff
@@ -0,0 +1,73 @@
+//Create an input and a button in html
+const button = document.getElementById("submit");
+const result = document.getElementById("result");
+const timer = document.getElementById("timer");
+let countS = 0;
+let countL = 0;
+const sCount = document.querySelector(".s-count");
+const lCount = document.querySelector(".l-count");
+
+
+button.addEventListener("click", () => {
+    let timer = document.getElementById("time").value;
+    //Check valid input
+    if (timer <= 0) {
+        return alert("Insert a valid time")
+    }
+
+    let res = "";
+    result.innerHTML = "";
+  
+  //Create an event listener
+    document.addEventListener("keydown", keyUser)
+    restartGame();
+    countdown(timer);
+    setTimeout(() => {
+        if (countS > countL) {
+            res = "S is the winner";
+        } else if (countS < countL) {
+            res = "L is the winner";
+        } else if (countS === 0 && countL === 0) {
+            res = "Key not pressed"
+        } 
+        result.innerHTML = res;
+        document.removeEventListener("keydown", keyUser)
+    }, timer * 1000);
+});
+
+//keypressed
+let keyUser = (event) => {
+    let keyChar = event.key
+    if (keyChar === "s") {
+        countS++;
+        showActive(".s-container");
```

Comment:
I'm getting error in console 
`Uncaught ReferenceError: showActive is not defined at HTMLDocument.keyUser `
and I don't see the counting for pressed button

Reply:
I'm not getting an error, everything works well for me in the codepen. There is also the counting for the pressed button in the code below :)